packages/cli-kit/src/public/node/themes/api.ts

```diff
@@ -353,6 +354,17 @@ export async function metafieldDefinitionsByOwnerType(type: MetafieldOwnerType,
   }))
 }
 
+export async function passwordProtected(session: AdminSession): Promise<boolean> {
```

Comment:
Why not keep the name of this the same? The previous seemed more specific with less room for misinterpretation.